cmd/minishift/cmd/config/validations.go

```diff
@@ -38,7 +38,9 @@ func IsValidDriver(string, driver string) error {
 }
 
 func RequiresRestartMsg(string, string) error {
-	fmt.Fprintln(os.Stdout, "To apply the changes, you must delete the current VM with `minishift delete` and start a new VM with `minishift start`.")
+	fmt.Fprintln(os.Stdout, "To apply the changes, "+
+		"you must delete the current VM with `minishift delete` if VM exist "+
```

Comment:
So is it safe to review this message now, or will we be changing the command itself to first check if the vm exists?